2023/03/identifying-vulnerabilities-using-horusec.md

```diff
@@ -0,0 +1,112 @@
+---
+author: "Indra Pranesh Palanisamy, Kürşat Kutlu Aydemir and Edgar Mlowe"
+title: "Identifying Vulnerabilities in Code using Horusec"
+featured:
+  image_url: /blog/2023/03/identifying-vulnerabilities-using-horusec/pexels-indra-pranesh-palanisamy-15837790.webp
+description: "In this blog post, we explore how Horusec, an open-source tool for identifying security vulnerabilities in code, can help developers improve the security of their applications by detecting potential threats and providing actionable insights. Learn how to integrate Horusec into your development process and enhance your code's security."
+date: 2023-03-07
+tags:
+- code-scan
+- security
+- vulnerability
+---
+
+![The sun has just started to rise over the horizon, casting a warm orange glow over the calm sea. A group of fishermen can be seen pushing their small wooden boats into the water, preparing to start their day's work. The air is still cool and quiet, with only the sound of waves gently lapping against the shore as the fishermen row out into the open sea, ready to begin their early morning fishing expedition.](/blog/2023/03/identifying-vulnerabilities-using-horusec/pexels-indra-pranesh-palanisamy-15837790.webp)
+
+<!-- Photo by Pranesh, 2022 -->
+
+## Horusec
+
+[Horusec](https://horusec.io/site/) is an open source tool that orchestrates other security tools and identifies security flaws or vulnerabilities in projects during the development process and puts all results in a database for analysis and generation of metrics.
+
+Currently, Horusec supports - C#, Java, Kotlin, Python, Ruby, Golang, Terraform, Javascript, Typescript, Kubernetes, PHP, C, HTML, JSON, Dart, Elixir, Shell, Nginx, Swift.
+
+It can also be integrated with CI/CD to execute the scan every time a developer raises a merge request.
+
+### Horusec CLI Installation
+
+#### Requirements
+
+- Docker
+- Git
+
+```
+curl -fsSL https://raw.githubusercontent.com/ZupIT/horusec/main/deployments/scripts/install.sh | bash -s latest
+```
+
+### VS Code Extension
+
+Horusec has a [VS code extension](https://docs.horusec.io/docs/extensions/visual-studio-code/) which is super helpful making complete code analysis with a single click.
```

Comment:
Can you show a screenshot of that? That would make it easier for me to imagine using the VS Code extension.

Reply:
Sure, I have added a screenshot of the VS code extension where the code analysis can be started with a single click.